permissions.md

```diff
@@ -0,0 +1,34 @@
+| Hide Lab elements for non-admins                                                            |   ❌   |   ❌   |   ❌   |   ✅   |
+| Data dropdown                                                                               |   ❌   |  🔑[^1]| 🔑[^1] |   ✅   |
+| Double-map toggle                                                                           |   ❌   |  🔑[^1]| 🔑[^1] |   ✅   |
+| Simulations dropdown                                                                        |   ❌   |   ✅   |   ✅   |   ✅   |
+| Download sims                                                                               |   ❌   |   ✅   |   ✅   |   ✅   |
+| Scenarios dropdown                                                                          |   ❌   |   ✅   |   ✅   |   ✅   |
+| Bookmarks panel                                                                             |   ❌   |   ❌   |   ✅   |   ✅   |
+| Create scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit scenarios                                                                              |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Duplicate scenarios                                                                         |   ❌   |   ❌   |   ✅   |   ✅   |
+| Run sims                                                                                    |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete sims                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit themes                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Copy/Delete Attributes                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete owned bookmarks                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Remove schema edits                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Emulate permissions                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Export non-Simulation layers                                                                |   ❌   |   ❌   |   ❌   |   ✅   |
+| Configurer modal                                                                            |   ❌   |   ❌   |   ❌   |   ✅   |
+| Download all scenarios                                                                      |   ❌   |   ❌   |   ❌   |   ✅   |
+| Show archived filters                                                                       |   ❌   |   ❌   |   ❌   |   ✅   |
+| Archive/unarchive filters                                                                   |   ❌   |   ❌   |   ❌   |   ✅   |
```

Comment:
You can archive filters?  Like parcel filters?  I didn't know that was a feature we had

Reply:
My bad, I was referring to simulations.